test/models/deploy_test.rb

```diff
@@ -1,67 +1,101 @@
 require_relative '../test_helper'
 
-SingleCov.covered! uncovered: 40
+SingleCov.covered! uncovered: 38
 
 describe Deploy do
   let(:project) { projects(:test) }
   let(:user) { users(:deployer) }
   let(:user2) { users(:admin) }
   let(:stage) { stages(:test_staging) }
+  let(:deploy) { deploys(:succeeded_test) }
 
-  describe "#deploy_buddy" do
-    before { @deploy = create_deploy! }
+  describe "#summary" do
+    let!(:deploy) { create_deploy! }
 
-    describe "no buddy message at all" do
-      it "returns no buddy name when BuddyCheck is not enabled" do
-        BuddyCheck.stubs(:enabled?).returns(false)
-        @deploy.summary.must_match(/#{user.name}  deployed/)
-      end
+    it "shows no buddy" do
+      deploy.summary.must_equal "Deployer  deployed baz to Staging"
```

Comment:
will be rendered as html, so does not really matter (also was there before
..) :)

On Mon, Apr 18, 2016 at 5:04 PM, Roman Sandler <notifications@github.com>
wrote:

> In test/models/deploy_test.rb
> <https://github.com/zendesk/samson/pull/846#discussion_r60155396>:
>
> >
> > -    describe "no buddy message at all" do
> > -      it "returns no buddy name when BuddyCheck is not enabled" do
> > -        BuddyCheck.stubs(:enabled?).returns(false)
> > -        @deploy.summary.must_match(/#{user.name}  deployed/)
> > -      end
> > +    it "shows no buddy" do
> > +      deploy.summary.must_equal "Deployer  deployed baz to Staging"
>
> extra whitespace?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/samson/pull/846/files/4ed10d0d40b50ebaa31c9ca457c1c00cf61100fc#r60155396>
>